### ISSUE TITLE
Organization's name

### ISSUE THREAD

**Issue by camilavargasp** [nceas-coding-club/nceas-coding-club.github.io#3]
**Goal:** Rename the nceas-hacky-hour organization to something broader so all the different "community coding" initiatives live under one umbrella.

- Initially, we thought of "NCEAS Community Learning Resources", but that seems too wordy and easy to confuse with the learning hub.
- Julien suggested using "NCEAS coding recipes" - This used to be part of the NCEAS website, however, became outdated and was taken down.
- Daphne suggested NCEAS Coding Club and Data Nerds
- Alex suggested NCEAS Data Bytes

Great ideas!

**In summary**
- We could name the organization "NCEAS Coding Club. And we can have a page on the website for "Data Recipes", where we include the more cookbook-y stuff. And a page named "Byte Size", where we keep smaller posts or links to outside resources we think are good (Like the [NPR Short Wave](https://www.npr.org/podcasts/510351/short-wave) podcast).
